engine/schema/src/com/cloud/upgrade/dao/Upgrade453to460.java

```diff
@@ -30,12 +30,12 @@
 import com.cloud.utils.exception.CloudRuntimeException;
 import com.cloud.utils.script.Script;
 
-public class Upgrade452to460 implements DbUpgrade {
```

Comment:
I think 4.5.3 would be only a bugfix release that ought not involve any schema change. Since, 4.5.2 to 4.5.3 won't have any schema changes, adding a linear upgrade paths simplifies the paths. IMO, it's just stubs.